SUPPORT.md

```diff
@@ -66,9 +66,13 @@ are:
 - `{ name }-{ target }-v{ version }{ archive-suffix }`
 - `{ name }-{ version }-{ target }{ archive-suffix }`
 - `{ name }-v{ version }-{ target }{ archive-suffix }`
-- `{ name }-{ version }-{ target }{ archive-suffix }`
-- `{ name }-v{ version }-{ target }{ archive-suffix }`
```

Comment:
These were already in the list.

Reply:
So they were, thank you for catching that